docs/source/examples/shapes_images_positions.rst

```diff
@@ -56,8 +57,7 @@ Shapes, Images and Positions
             # Use PIL when you want to import an image from the web
             import requests
             from PIL import Image
-            img = Image.open(requests.get("https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png",
-                                          stream=True).raw)
+            img = Image.open("../logo/cropped.png")
```

Comment:
The way this is written will not work under Windows; you'd need to specify the path using `os.path` or `pathlib`.

It **should** (and this really is a rather big should) be true that sphinx-build executes the code within the `docs/source` directory, so the actual path would be `../../logo/cropped.png` -- but I would find it better if there were an absolute way to specify the path -- but I am not sure what is the best way to do so.

If you manage to find a good solution for this, I will be okay with it -- and otherwise I suggest we remove this example.

Reply:
I removed the example, maybe we will make it in a future pr